Jenkinsfile

```diff
@@ -23,14 +23,6 @@ properties(
 node {
   checkout scm
 
-  if (params.ARMORYSPINNAKER_JENKINS_JOB_ID != '') {
```

Comment:
this will be automatically passed into the tests, and the tests will fetch the correct job id